mod.d/bccmysqldqslower.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+ --- !ec2rlcore.module.Module
+# Module document. Translates directly into an almost-complete Module object
+name: !!str bccmysqldqslower
+path: !!str
+version: !!str 1.0
+title: !!str Collect output from mysqld_qslower for system analysis
+helptext: !!str |
+  Collect output from mysqld_qslower for system analysis
+  Requires --period= for length of sample period in seconds
+  Requires --threshold= for retrieving queries slower than the specified threshold (in ms)
+  mysqld_qslower is available from the bcc package https://github.com/iovisor/bcc
+  It requires a kernel that supports BPF, the relevant devel packages be installed,
+  debugfs mounted, and the tool in your PATH
+placement: !!str run
+package: 
+  - !!str bcc https://github.com/iovisor/bcc
+language: !!str bash
+content: !!str |
+  #!/bin/bash
+  error_trap()
+  {
+      printf "%0.s=" {1..80}
+      echo -e "\nERROR:	"$BASH_COMMAND" exited with an error on line ${BASH_LINENO[0]}"
+      exit 0
+  }
+  trap error_trap ERR
+
+   # read-in shared function
+  source functions.bash
+
+   echo "I will collect mysqld_qslower output from this $EC2RL_DISTRO box for $period seconds."
+
+   timeout --preserve-status $period mysqld_qslower `pgrep -n mysqld` $threshold
+
+   exit 0
+constraint:
+  requires_ec2: !!str False
+  domain: !!str net
+  class: !!str performance
+  distro: !!str alami alami2 ubuntu rhel suse
+  required: !!str period times threshold
```

Comment:
Need to remove 'times' here too.

Reply:
Updated CHANGELOG and edited the module to remove "times" requirement

Reply:
Looks good!